src/parsers/manifestjson.js

```diff
@@ -478,6 +479,20 @@ export default class ManifestJSONParser extends JSONParser {
         });
       }
     }
+
+    if (this.parsedJSON.name) {
+      const nameLowerCase = String(this.parsedJSON.name).toLowerCase();
```

Comment:
This should already be a string, no? Can you explain why casting to `String` again is necessary?

Reply:
Sure @wagnerand , i made the parse to String because it wasn't recognizing the `toLowerCase` as a function like the image below. is there a best way to do it?

<img width="592" alt="Screen Shot 2020-04-17 at 10 49 04" src="https://user-images.githubusercontent.com/648079/79580264-265ce900-809f-11ea-9577-6729ef982096.png">

Reply:
So, this comes from the test `should collect an error on non-string name value`. It makes sure we fail if the name is not a string. In that case, we don't need to check whether it contains any of the forbidden words.

You would want to check whether the name is a string as well:

```
if (this.parsedJSON.name && typeof this.parsedJSON.name === 'string') {
```